www/static/js/docs.js

```diff
@@ -17,19 +17,30 @@
 
 $(document).ready(function () {
 
-    function slugify(text) {
+    function slugifyLikeGitHub(originalText) {
+
+        var text = originalText;
+
+        // convert to lowercase
         text = text.toLowerCase();
 
-        // replace unaccepted characters with spaces
+        // replace whitespace characters with dashes
+        text = text.replace(/\s/g, '-');
+
+        // remove dashes at the start and end if they weren't there since the start
```

Comment:
These are the results from GitHub:

    "  -hello" => "---hello"
    "   hello" => "---hello"

That is, the truncation actually never happens. Fixing now.